test/room-test.js

```diff
@@ -16,13 +16,13 @@ describe('Room', () => {
   it('should have a number', () => {
```

Comment:
Again, I'm not sure if it's really worth trying, but here you could find a way to make sure that if a data set doesn't have a room number (for some weird reason), that it's doesn't break your code.

Reply:
Another great catch! I'm wondering if that is handled when the api checks the data that is being posted, but I will definitely look into how to make sure that that scenario is being handled somewhere.